generic-server/src/main.rs

```diff
@@ -4,6 +4,13 @@ use std::net::SocketAddr;
 
 #[tokio::main]
 async fn main() {
+
+    // Load dotenv-flow variables
+    dotenv_flow::from_filename(".env.example").ok();
```

Comment:
The code must not depend on `.env.example`. Each one will have to create a `.env` file on its computer; it is fine. Remember to add `.env` to `.gitignore`.
```suggestion
    dotenv_flow::dotenv_flow().ok();
```

Reply:
Hello

Reply:
There are also conflicts to resolve.

Reply:
Okay i am just from correcting it

Reply:
I still see `.env.example` in the code.